spec/inputs/ganglia_spec.rb

```diff
@@ -56,5 +56,10 @@
       expect(event["tmax"]).to eq(60)
     end
 
+    it "should receive the name and value from ganglia" do
+      expect(event["value"]).to eq(7000)
```

Comment:
Is a good practice to have a check per test, I recommend you split this into two different test examples.  This provides a better maintainability over the time for the test suite.